### ISSUE TITLE
Implement extended annotations statistics

### ISSUE THREAD

**Issue by AYZemlyakov** [opencv/cvat#1783]
### My actions before raising this issue
- [ ] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [ ] Searched [past issues](/issues)

You need to implement extended annotation statistics, including the following features:

- Download statistics as CSV file.
- Add attributes of labels to output statistics (summarised as number of object, of course)
- Add nodes (number of points if we speak about polygon) to output statistics
- Add a new action to "Task" entity (download statistics over all jobs)

### Steps to Reproduce (for bugs)

1.
1.
1.
1.

### Context
We are a computer vision company and we use CVAT all the time. We definitely need more detailed statistics than we have now. Out typical annotation team is about 20-50 people, each of them annotate their own batches/tasks/jobs and we need detailed statistics including what I have described above. For example, we have 10 classes and about 25 attributes but when I download statistics I can see only classes/objects but I can't see attributes and that's very bad. Also, it's very uncomfortable not being able to download statistics as CSV files.

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by s3298230** [comment]
Would it be possible to track the assignees (include  their working time) on each job? It will be helpful for task audit

**Comment by APeFerreira** [comment]
Are there any updates on this topic? It would be really insightful to download a json or csv of jobs/tasks annotation statistics, as we can manually see when opening the **Info** tab of a given job, via the web page.

**Comment by zhiltsov-max** [comment]
@APeFerreira, yes, analytics has been reworked significantly and now provides statistics for annotations and other objects. Please try to check task Actions > View Analytics. Most of the functionality for this feature is available on https://app.cvat.ai as a paid option now.

<img width="200" height="365" alt="Image" src="https://github.com/user-attachments/assets/7891ad11-253f-482b-937e-aef85be08cd3" />

Annotations:
<img width="2340" height="635" alt="Image" src="https://github.com/user-attachments/assets/d2e39a2c-148a-4154-9999-decf5cd2078b" />

Workflow events:
<img width="1000" height="587" alt="Image" src="https://github.com/user-attachments/assets/0eb819d3-c044-476b-be60-70208fd0c0ae" />

You can download a csv file with the displayed table using the button at the top of each table:

<img width="59" height="81" alt="Image" src="https://github.com/user-attachments/assets/6ee03fce-7fc6-406f-88d4-a2f117b1db22" />

**Comment by APeFerreira** [comment]
@zhiltsov-max thank you for the quick answer. I'm self-hosting CVAT, so I can't access that unless I opt for one of the paid plans, right?

**Comment by zhiltsov-max** [comment]
@APeFerreira, free self-hosted versions only allow downloading a list of events.

**Comment by nmanovic** [comment]
I will close the issue. Analytics is available only on CVAT Online (https://www.cvat.ai/pricing/cvat-online) and CVAT Enterprise (https://www.cvat.ai/pricing/enterprise). Use events to calculate it manually. They have all required information. 

https://docs.cvat.ai/docs/qa-analytics/analytics/
